packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -104,17 +111,60 @@ export class ProjectService {
         return project;
     }
 
-    async create(user: SessionUser, data: CreateProject): Promise<Project> {
+    async create(user: SessionUser, data: CreateProject): Promise<string> {
         if (user.ability.cannot('create', 'Project')) {
             throw new ForbiddenError();
         }
-        const adapter = await ProjectService.testProjectAdapter(data);
-        const explores = await adapter.compileAllExplores();
-        const projectUuid = await this.projectModel.create(
-            user.organizationUuid,
-            data,
+
+        const jobUuid = await this.startJob(undefined);
+
+        this.createProject(user, data, jobUuid);
```

Comment:
beautiful, so this kicks off a background job basically. But this is breaking the api.

But I think the approach is great